tls/s2n_x509_validator.c

```diff
@@ -701,6 +708,23 @@ static S2N_RESULT s2n_x509_validator_verify_cert_chain(struct s2n_x509_validator
      */
     X509_STORE_CTX_set_flags(validator->store_ctx, X509_V_FLAG_PARTIAL_CHAIN);
 
+#ifdef S2N_LIBCRYPTO_SUPPORTS_CUSTOM_OID
+    /* Custom critical oids are only supported when AWSLC_API_VERSION >= 34.
+     * See https://github.com/aws/aws-lc/pull/2426
+     */
```

Comment:
I'm not sure this comment is very useful currently. We have a feature probe, so the aws-lc API version doesn't matter really at the moment. This comment will be good to add for when the feature probe is removed, though.